packages/@secretlint/types/src/SecretLintRuleTranslator.ts

```diff
@@ -14,24 +14,35 @@ export type SecretLintRuleLocaleTag =
     | "zh-CN"
     | "zh-TW"
     | string;
+export type SecretLintRuleLocalizeMessageProps = { [index: string]: any };
+
+export type SecretLintRuleLocalizeMessageHandler<Props extends SecretLintRuleLocalizeMessageProps> = (
+    props?: Props
+) => string;
+export type SecretLintRuleLocalizeMessageMulti<Props extends SecretLintRuleLocalizeMessageProps> = {
+    [P in SecretLintRuleLocaleTag]: SecretLintRuleLocalizeMessageHandler<Props>;
+} & {
+    // Workaround for https://github.com/YousefED/typescript-json-schema/issues/110
+    en(props?: Props): string;
```

Comment:
https://github.com/YousefED/typescript-json-schema/issues/110 does not support `function`.
It should be ignored.